verify/cancel.py

```diff
@@ -0,0 +1,27 @@
+import os
+from os.path import join, dirname
+import sys
+from dotenv import load_dotenv
+
+dotenv_path = join(dirname(__file__), '../.env')
+load_dotenv(dotenv_path)
+
+NEXMO_API_KEY = os.getenv('NEXMO_API_KEY')
+NEXMO_API_SECRET = os.getenv('NEXMO_API_SECRET')
+
+if len(sys.argv) != 2:
```

Comment:
If you need to parse command-line parameters, I find `argparse` is usually just as compact, more understandable, and provides a better UX than manually extracting tokens from `sys.argv`.

I'm happy to port this to argparse if you're busy with other things.